docs/self-managed/concepts/multi-tenancy.md

```diff
@@ -0,0 +1,86 @@
+
+- [Identity feature flags](../../../self-managed/identity/deployment/configuration-variables/#feature-flags)
+- [Zeebe multi-tenancy](../../../self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy)
+- [Operate multi-tenancy](../../../self-managed/operate-deployment/operate-configuration/)
+- [Tasklist multi-tenancy](../../../self-managed/tasklist-deployment/tasklist-configuration/#multi-tenancy)
+- [Optimize multi-tenancy]($optimize$/self-managed/optimize-deployment/configuration/multi-tenancy/)
+- [Connectors multi-tenancy](../../../self-managed/connectors-deployment/connectors-configuration/#multi-tenancy)
+  :::
+
+## Multi-tenancy in Camunda 8
+
+Multi-tenancy in the context of Camunda 8 refers to the ability of the Camunda 8 platform to serve multiple distinct
+tenants or clients within a single installation. Multi-tenancy in Camunda 8 extends these capabilities to cater to the
+needs of different departments, teams, or even external clients, all within a shared Camunda environment. Here's a closer
+look at what multi-tenancy is in Camunda 8:
+
+### Isolation of Data and Processes
+
+In a multi-tenant Camunda 8 installation, each tenant's data and processes are logically isolated from one another.
+This means that one tenant's workflows, data models, and process configurations do not interfere with or impact the
+operations of other tenants. Each tenant operates in a separate and secure space within the same Camunda 8 instance.
+
+### Resource Sharing
+
+Despite the isolation, multi-tenancy in Camunda 8 allows for efficient resource sharing. Tenants can leverage the same
+Camunda 8 software installation, reducing infrastructure costs and resource overhead. This shared model optimizes
+resource utilization and ensures that the platform remains cost-effective.
+
+### Efficient Administration
+
+Administrators can manage all tenants from a centralized administration console. This simplifies the process of
+monitoring and maintaining different tenant environments, making administrative tasks more efficient and reducing
+overhead.
+
+### Security
+
+Security is a paramount concern in multi-tenant Camunda 8 installations. Robust access control mechanisms ensure that
+tenants cannot access each other's data or processes. Security measures are in place to maintain the privacy and
+integrity of each tenant's information.
+
+### Cost-Effective
+
+Multi-tenancy in Camunda 8 is particularly beneficial for organizations that want to offer BPM and workflow automation
+capabilities to different departments or clients without the need for separate Camunda installations. This
+cost-effective approach minimizes infrastructure and maintenance costs.
+
+## How does it work?
+
+Camunda 8 implements multi-tenancy by relying on tenant identifiers in a single Camunda 8 installation. The data of
+all tenants is stored in a single data warehouse. Isolation is provided by appending a tenant identifier to each data
```

Comment:
why data warehouse?

Reply:
I was trying use a general term while explaining that all tenants share the same data storage, the same as the Camunda 8 installation. @felix-mueller do you think "data storage" would fit better?